app/keepers/keepers.go

```diff
@@ -33,6 +33,10 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/upgrade"
 	upgradekeeper "github.com/cosmos/cosmos-sdk/x/upgrade/keeper"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	icq "github.com/strangelove-ventures/async-icq/v4"
+	icqtypes "github.com/strangelove-ventures/async-icq/v4/types"
```

Comment:
What was the reasoning for reordering here? Seems like this PR should have nothing to do with icq

Reply:
This was caused due to fixing merge conflict then go fmt, unfortunately I do not know the exact reason why or how this works, but go fmt would automatically make this change since I have fixed merge conflict :(